src/main/java/org/sonar/plugins/web/checks/structure/ChildElementIllegalCheck.java

```diff
@@ -58,7 +58,10 @@ public void startElement(TagNode element) {
           createViolation(childNode.getStartLinePosition(), "Remove this \"" + child + "\" tag; it is not a legal child of \"" + parent + "\".");
         }
       }
+    } else if (element.equalsElementName(parent)) {
+
     }
+    int BAD_NAME;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "BAD_NAME" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)